mc/client-s3.go

```diff
@@ -158,27 +157,40 @@ func (w *watchObject) Close() {
 	close(w.done)
 }
 
-func (c *s3Client) Watch(recursive bool) (*watchObject, *probe.Error) {
-	eventChan := make(chan Event)
-	errorChan := make(chan *probe.Error)
-	doneChan := make(chan bool)
+// Enable/Disable lambda bucket notification
+func (c *s3Client) ToogleLambdaNotificationStatus(enable bool, region, accountID string) *probe.Error {
 
 	bucket, _ := c.url2BucketAndObject()
 
-	// todo(nl5887): correct arn creation
-	accountARN := minio.NewArn("minio", "lambda", "us-east-1", "mc", "lambda")
+	if err := isValidBucketName(bucket); err != nil {
+		return err
+	}
 
-	// enable bucket notifications
-	lc := minio.NewNotificationConfig(accountARN)
-	lc.AddEvents(minio.ObjectCreatedAll, minio.ObjectRemovedAll)
+	accountARN := minio.NewArn("minio", "lambda", region, accountID, "lambda")
 	mb := minio.BucketNotification{}
-	mb.AddLambda(lc)
+	if enable {
+		// enable bucket notifications
+		lc := minio.NewNotificationConfig(accountARN)
+		lc.AddEvents(minio.ObjectCreatedAll, minio.ObjectRemovedAll)
+		mb.AddLambda(lc)
+	}
```

Comment:
the toggle should remove only the aforementioned lambda config not the entire bucket notification we shouldn't edit other configs.